### PR TITLE
Some documentation improvements and added frontend documentation

### PR DESCRIPTION
**Pull request by KostLinux**
# Description

Documentation improvements for https://github.com/LookinLabs/GoBlitz/issues/20

Fixes # (issue)

## Dependencies

- GoBlitz

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- Via local development environment

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas (optional)
- [x] I have made corresponding changes to the documentation (optional)
- [x] My changes generate no new warnings
- [x] I've performed/added tests/lints that prove my fix is effective or that my feature works (optional)

# Pull requests related:

- https://github.com/LookinLabs/GoBlitz/pull/22
- https://github.com/LookinLabs/GoBlitz/pull/21

Before creating pull request, look that all points under checklist are done!

### CODE REVIEW

File: docs/introduction/folder_structure.md [hunk 1]
```diff
@@ -7,15 +7,12 @@ GoBlitz Web Framework has the following folder structure:
 
 - `bin` - Binary folder mostly used for different scripts for contribution
 - `controller` - The Controller component of MVC architecture. Responsible of receiving the request and sending the response while communicating with the model and repository layer
+- `helperes` - The Helpers folder for the helper functions
```
Comment (Matrix278) [inline docs/introduction/folder_structure.md:10]: I guess something wrong with this word here

**Comment by akinakk** [comment]
Great job on enhancing the documentation especially with the addition of sessions documentation

File: docs/sessions/sessions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The controller uses model to check the user existence by getting a boolean value. If the user exists, the controller returns an error message. If the user doesn't exist, the controller creates a record in database about the user and sets the session.
+
+## Signin
+
+On the signin the controller renders data from `loginFormData` struct and compares with the data in database. 
+
+The controller uses model to check for the user existance and compares the password input with the encrypted record in database.
+
+If the user exists and the password is correct, the controller sets the session. If the user doesn't exist or the password is incorrect, the controller returns an error message.
```
Comment (Matrix278) [inline docs/sessions/sessions.md:22]: I would use two words for the sign up and sign in, as it's a two-word verb phrase, or at least in the text sign-in, sign-up.

**Review by Matrix278 — COMMENTED** [review]
@KostLinux Left one comment

**Review by Matrix278 — APPROVED** [review]

File: docs/sessions/sessions.md (hunk 2 above)
Comment (Matrix278) [reply docs/sessions/sessions.md:22]: You decided not to change it?

**Pull request merged**